content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,43 @@ environment, then Prometheus is a good choice.
 
 ### Architecture
 
-Sensu has a [number of components](https://docs.sensu.io/sensu-core/latest/overview/architecture/). It uses
-RabbitMQ as a transport, Redis for current state, and a separate server for
-processing and API access.
+Sensu is a primarily agent-based monitoring solution ([reference](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/deployment-architecture/)). Sensu agents automatically connect to one or more Sensu backends over an encrypted pub/sub transport (TLS-encrypted websockets), and handle automated registration/de-registration of entities and local service discovery. Sensu [backends](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/) provide the control plane, including the configuration [API](https://docs.sensu.io/sensu-go/latest/api/) and [scheduler](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/). 
 
-All components of a Sensu deployment (RabbitMQ, Redis, and Sensu Server/API) can be clustered for highly available and redundant configurations.
+All components of a Sensu deployment can be clustered for high availability and improved event-processing throughput. Sensu also provides built-in support for federation (including RBAC and other API resource replication) which is recommended for deployments larger than 30,000 nodes (or entities) under management.
 
 ### Summary
-If you have an existing Nagios setup that you wish to scale as-is, or want to take advantage of the automatic registration feature of Sensu, then Sensu is a good choice.
 
-If you want to do whitebox monitoring, or have a very dynamic or cloud based environment, then Prometheus is a good choice.
+Sensu and Prometheus have a few capabilities in common, but they take very different approaches to monitoring. Both offer extensible discovery mechanisms for dynamic cloud-based environments and ephemeral compute platforms (though the underlying mechanisms are quite different). Both provide support for collecting multi-dimensional metrics via labels (and annotations). Both have extensive integrations (and Sensu natively supports collecting metrics from all Prometheus exporters). Where Sensu and Prometheus differ the most is in their use cases.
+
+Where Sensu is better: 
+
+- If you're collecting and processing hybrid observability data (including metrics and events)
+- If you're monitoring hybrid infrastructure (including on-prem servers, public/private cloud, and cloud-native/Kubernetes)
+- If you're consolidating mulitple monitoring tools and need support for metrics _and_ traditional monitoring Nagios-style plugins or check scripts
+- If you are already invested in one or more data platforms like InfluxDB, Elasticsearch, Splunk, TimescaleDB, Wavefront, etc (Sensu's primary function as an observability pipeline means it is designed to route data to the data platform(s) of your choosing)
+- More powerful alerting and event-processing platform
```

Comment:
It's a fair question RE: alerts, and the differences RE: alerting are probably so nuanced that it's not worth mentioning here. There are far more similarities than differences when it comes to basic alerting (e.g. both offer "silencing", "grouping" is called "aggregation" in Sensu, and "inhibition" is called "dependency filtering" in Sensu), so I'll drop that mention. 

As an aside – Sensu Go does achieve a degree of "in-built alert generation" now by means of our Asset framework. In practice Sensu shares some concepts with the Kubernetes platform in that many of the "built-in" components (e.g. kube-dns) are packaged as containers (tarballs w/ metadata) and updated via K8s resource definitions which pull & run those containers on demand; in Sensu Go integrations are packaged as Assets (tarballs w/ metadata) and updated via Sensu resource definitions with pull & run those integrations on demand. If I understand how AlertManager works, I think the difference is that instead of having to ship new AlertManager releases to add new integrations or modify existing ones, we ship new Assets and update the resource definition templates so existing deployments can easily take advantage of new features. 

TL;DR - AlertManager offers more built-in integrations out of the box (a handful of hard-coded integrations, plus webhooks), but they still have to be configured. You need that same configuration in Sensu, but the Sensu config includes references to integrations that can be pulled and run on-demand; and OSS/third-party integrations can be added with the exact same UX.